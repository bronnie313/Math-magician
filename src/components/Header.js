const Header = () => (
  <nav>
    <div>Math Magicians</div>
    <ul>
      <li>Home</li>
      <li>Calculator</li>
      <li>Quote</li>
    </ul>
  </nav>
);

export default Header;
