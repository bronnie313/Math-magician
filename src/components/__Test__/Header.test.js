import React from 'react';
import { render, screen } from '@testing-library/react';
import Header from '../Header';

describe('Header component', () => {
  it('renders the header text', () => {
    render(<Header />);
    const headerText = screen.getByText('Math Magicians');
    expect(headerText).toBeInTheDocument();
  });

  it('renders the navigation items', () => {
    render(<Header />);
    const navItems = screen.getAllByRole('listitem');
    expect(navItems.length).toBe(3);
  });
});
