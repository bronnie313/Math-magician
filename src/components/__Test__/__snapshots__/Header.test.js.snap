// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component component renders correctly 1`] = `
<nav>
  <div>
    Math Magicians
  </div>
  <ul>
    <li>
      Home
    </li>
    <li>
      Calculator
    </li>
    <li>
      Quote
    </li>
  </ul>
</nav>
`;
